proposals/extensions.md

```diff
@@ -292,12 +292,21 @@ Adjust scoping rules so that type parameters are in scope within the 'for':
 implicit extension E<T> for T { }
 ```
 
-## Open issue: need to specify conversion rules
+## Open issue: need to specify user-defined conversion rules
 
 ```
-explicit extension R for U { } 
-R r = default;
-object o = r; // what conversion is that? if R doesn't have `object` as base type. What about interfaces?
+implicit extension EU for U
+{
+    public static implicit operator EU(int i) => ...;
+}
+implicit extension EInt for int { }
+
+R r = 42;
+U u = 43;
+
+EInt ei = 44;
+R r = ei;
+U u = ei;
```

Comment:
You have renamed the `R` extension to `EU` in the declaration, but not at the usage location :

```suggestion
implicit extension EU for U
{
    public static implicit operator EU(int i) => ...;
}
implicit extension EInt for int { }

EU eu = 42;
U u = 43;

EInt ei = 44;
EU eu = ei;
U u = ei;
```